packages/core/src/Modules/Cashier/Components/Form/account-transfer-form.jsx

```diff
@@ -362,4 +370,6 @@ export default connect(({ modules, ui }) => ({
     transfer_fee: modules.cashier.config.account_transfer.transfer_fee,
     transfer_limit: modules.cashier.config.account_transfer.transfer_limit,
     is_nativepicker_visible: ui.is_nativepicker_visible,
```

Comment:
I don't think we need this anymore if we just use the `Desktop` and `Mobile` wrappers to render either `Dropdown` or `SelectNative`.